README.md

```diff
@@ -175,8 +175,10 @@ err := grace.Wait(
   10*time.Second,
   grace.WithWaitForTCP("localhost:6379"), // redis
   grace.WithWaitForTCP("localhost:5432"), // postgres
+  grace.WithWaitForUnix("unix:///tmp/something.sock"), // something on a unix socket
```

Comment:
Nit: I think these docs are missing the most recent update

Reply:
Whoops. Fixed.